chronosphere/resource_resource_pools_config.go

```diff
@@ -246,7 +246,9 @@ func validateResourcePoolsConfig(cfg *apimodels.Configv1ResourcePools) error {
 			return err
 		}
 	}
-	if sum != 100 {
+	// We support allocations up to three decimal places.
+	// TODO(codyg): dry run validation to avoid duplicating this logic.
+	if sum < 99.999 || sum > 100.001 {
```

Comment:
I don't quite follow, doesn't this mean you could end up w/ 99.9999 for example? Or is that not possible since we're restricting individual allocations to three decimal places

Reply:
Anything between [99.999, 100.001] (inclusive) is rounded to 100 server-side.